Form.html

```diff
@@ -0,0 +1,27 @@
+<doctype! html>
+<html>
+<head>
+<title> Form Template </title>
+</head>
+<body>
+<h2> Please provide the following information </h2>

```

Comment:
As it's heading, make it h1.

Reply:
It's personal preference, you and everyone is free to change it to anything. If need to edit the h2, then you can download and edit the file yourself. It's just a template.

Thanks